dockerfiles/che/Dockerfile

```diff
@@ -5,13 +5,22 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #
-FROM openjdk:8u242-jre-slim
+
+# Variables in `COPY --from=` is not supported see https://github.com/moby/moby/issues/34482
+# this is workaround to handle that.
+ARG CHE_DASHBOARD_VERSION=next
+ARG CHE_WORKSPACE_LOADER_VERSION=next
+
+FROM quay.io/eclipse/che-dashboard:${CHE_DASHBOARD_VERSION} as che_dashboard_base
```

Comment:
3 FROMs in a row may not work in CRW because I don't think this works in Brew.

Reply:
If it does not work then the whole thing doesn't make sense.

Reply:
finally got a build to run in Brew... I did this:

```
FROM jboss-eap-7/eap73-openjdk11-openshift-rhel8 as EAP73
FROM jboss-eap-7/eap72-openjdk11-openshift-rhel8 as EAP72
FROM ubi8-minimal:8.2

USER root
RUN microdnf install -y diffutils
COPY --from=EAP73 /opt/jboss/container/wildfly/s2i/galleon/build-image-settings.xml /tmp/
COPY --from=EAP73 /opt/eap/bin/openshift-launch.sh /tmp/eap73-launch.sh
COPY --from=EAP72 /opt/eap/bin/openshift-launch.sh /tmp/eap72-launch.sh
RUN ls -la /tmp; diff -u /tmp/eap7*.sh || true
```

And it succeeded!

http://download.eng.bos.redhat.com/brewroot/work/tasks/5422/28455422/x86_64.log